replace_attrs.py

```diff
@@ -212,7 +212,8 @@ def prettify_output(html):
             else:
                 confirm = 'y'
             if confirm.lower()[0] == 'y':
-                with open(xml_file, 'wb') as rf:
+                new_file_name = f"{xml_file.rsplit('.', 1)[0]}_v17migrated.{xml_file.rsplit('.', 1)[1]}"
```

Comment:
[comment]
```suggestion
                new_file_name = "_v17migrated.".join(xml_file.rsplit(".", 1))
```

Reply:
**[change]** I would remove v17 from the new name, maybe this will be also needed for v18 and next ones when migrating from older versions. _migrated would be enough imho